manticore/models.py

```diff
@@ -0,0 +1,80 @@
+from .utils.helpers import issymbolic
+from .core.smtlib.solver import solver
+from .core.smtlib.operators import ITEBV, ZEXTEND
+
+
+def _find_zero(cpu, constrs, ptr):
```

Comment:
does this belong elsewhere? cpu? platform (we already have _read_string there too)?